tests/fixtures/importer/units.csv

```diff
@@ -1,8 +1,8 @@

```

Comment:
I updated this test fixture to reflect the change in schema noted in this table:

<img width="689" alt="Screen Shot 2021-02-02 at 11 46 59 AM" src="https://user-images.githubusercontent.com/12176173/106641038-6321a000-654c-11eb-82ae-4213717c2ad3.png">